src/NCPopupMenu.cc

```diff
@@ -107,11 +136,13 @@ bool NCPopupMenu::postAgain()
     // yuiDebug() << "Index: " << selection << std::endl;
     YTableItem * tableItem = dynamic_cast<YTableItem *>( getCurrentItemPointer() );
 
-    YMenuItem * item = itemsMap[ tableItem ];
+    auto current = find_item(tableItem);
```

Comment:
Please don't defeat the strict type checking of C++ with declaring so many variables as `auto`. Please be explicit.